exp/security.md

```diff
@@ -27,20 +27,22 @@ The following table shows the authentication methods that are in place for the d
 
-Anbox Cloud uses secure and isolated system containers supplied by [LXD](https://ubuntu.com/lxd). LXD provides a high degree of flexibility when setting up containers, allowing you to run in a fully secure or less secure way, depending on your requirements. See [Security](https://documentation.ubuntu.com/lxd/en/latest/security/) in the LXD documentation for more information about how a LXD setup can be secured.
+Anbox Cloud uses secure and isolated system instances supplied by [LXD](https://ubuntu.com/lxd). LXD provides a high degree of flexibility when setting up instances, allowing you to run in a fully secure or less secure way, depending on your requirements. See [Security](https://documentation.ubuntu.com/lxd/en/latest/security/) in the LXD documentation for more information about how a LXD setup can be secured.
+
+Using virtual machines to host Android containers provides provides better workload isolation.
 
 Anbox Cloud uses LXD in a way that enforces the highest security level.
 
-### Unprivileged containers
+### Unprivileged instances
 
-Many container managers use privileged containers, which means that the containers have root privileges on the host system, including access to all devices. This is a security risk, because attackers could gain control over the host system.
+Many instance managers use privileged instances, which means that the instances have root privileges on the host system, including access to all devices. This is a security risk, because attackers could gain control over the host system.
 
-Anbox Cloud uses unprivileged LXD containers only, which fully isolates the containers and ensures that they cannot gain root privileges. In addition, the Android container that runs inside the LXD container also runs as an unprivileged container. This method isolates the Android container twice, with the result that if the encapsulation of either the LXD container or the Android container should fail, the system would still be protected.
+Anbox Cloud uses unprivileged LXD instances only, which fully isolates the instances and ensures that they cannot gain root privileges. In addition, the Android container that runs inside the LXD instance also runs as an unprivileged instance. This method isolates the Android container twice, with the result that if the encapsulation of either the LXD instance or the Android container should fail, the system would still be protected.
 
 [note type="caution" status="Warning"]
-While containers are fully isolated, all containers currently use the same GPU resources. As a result, any container that is launched with GPU support could take all GPU resources in a DDoS-like attack, which would prevent other containers from starting.
+While instances are fully isolated, all instances currently use the same GPU resources. As a result, any instance that is launched with GPU support could take all GPU resources in a DDoS-like attack, which would prevent other instances from starting.
```

Comment:
yes, I understood that. But the usage of instance is just a generalization that we are attempting throughout the documentation. All we are claiming in this section is that our instances are unprivileged which is true, whether it is a container (because we have taken measures to implement unprivileged containers) or is a VM (becase VM is itself unprivileged). Does it make sense?

Reply:
Sure, thanks!